### PR TITLE
[NEW] Video support

### LINKED ISSUE #712: [NEW] Record Video

### PR DESCRIPTION
**Pull request by pranavpandey1998official**
@RocketChat/ReactNative

Closes #712

### CODE REVIEW

**Comment by pranavpandey1998official** [comment]
@diegolmello I have done the changes you should check on IOS

**Comment by djorkaeffalexandre** [comment]
Working nice on iOS.

**Comment by djorkaeffalexandre** [comment]
@pranavpandey1998official I added pt-BR translate and remove log:
https://github.com/pranavpandey1998official/Rocket.Chat.ReactNative/pull/5

File: app/containers/MessageBox/UploadModal.js [hunk 1]
```diff
@@ -166,9 +185,26 @@ export default class UploadModal extends Component {
...
 	}
 
+	renderPreview() {
+		const { file } = this.props;
+		if (file.mime && file.mime.match(/image/)) {
+			return (<Image source={{ isStatic: true, uri: file.path }} style={styles.image} />);
+		}
+		if (file.mime && file.mime.match(/video/)) {
+			return (
+				<View
```
Comment (diegolmello) [inline app/containers/MessageBox/UploadModal.js]:
```jsx
<View style={styles.video}>
```

File: app/containers/MessageBox/index.js [hunk 2]
```diff
@@ -482,6 +495,7 @@ class MessageBox extends Component {
 	}
 
 	takePhoto = async() => {
+		console.warn('take photo');
```
[the remark this answers is not in the document]
Comment (diegolmello) [reply app/containers/MessageBox/index.js]: @pranavpandey1998official

**Pull request merged**